package.json

```diff
@@ -44,29 +43,9 @@
     "string-argv": "0.3.1",
     "supports-color": "9.0.2"
   },
-  "devDependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/eslint-parser": "7.16.3",
-    "@babel/preset-env": "^7.16.0",
-    "babel-jest": "27.3.1",
-    "consolemock": "^1.1.0",
-    "eslint": "^8.2.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "fs-extra": "^10.0.0",
-    "husky": "^7.0.4",
-    "jest": "^27.3.1",
-    "jest-mock-console": "1.2.3",
-    "jest-snapshot-serializer-ansi": "^1.0.0",
-    "prettier": "^2.4.1"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
-  },
+  "workspaces": [
```

Comment:
Maybe let's document how tests are supposed to be run now?

Reply:
With `npm >= 7` it's as simple as `npm install && npm test`. This way the dev dependencies are excluded from the published package.json, but still installed into the root `node_modules` locally.

But yeah... maybe I'll omit this commit since it's not strictly necessary.

Reply:
Removed the commit that moved them away.